# Define defaults for Solaris.
#
# SSH_SOURCE_BASHRC needs to be enabled for compatibility between SunSSH and
# OpenSSH. Bash needs to detect whether it's running in the SSH command mode in
# order to source .bashrc. SunSSH uses sockets for communication which is
# detected via getpeername. OpenSSH uses pipes which is not detected by
# getpeername. Enabling SSH_SOURCE_BASHRC makes bash also look at SSH_CLIENT
# and SSH2_CLIENT environment variables in addition to getpeername. The
# environment variables are set by OpenSSH.
#
# Not for upstream.

--- config-top.h	2018-10-12 21:35:23.000000000 +0200
+++ config-top.h	2019-02-06 15:05:56.837209396 +0100
@@ -63,7 +63,7 @@
 /* The default value of the PATH variable. */
 #ifndef DEFAULT_PATH_VALUE
 #define DEFAULT_PATH_VALUE \
-  "/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:/sbin:."
+  "/usr/gnu/bin:/bin:/usr/bin:/sbin:/usr/sbin:."
 #endif
 
 /* If you want to unconditionally set a value for PATH in every restricted
@@ -74,7 +74,7 @@
    the Posix.2 confstr () function, or CS_PATH define are not present. */
 #ifndef STANDARD_UTILS_PATH
 #define STANDARD_UTILS_PATH \
-  "/bin:/usr/bin:/sbin:/usr/sbin:/etc:/usr/etc"
+  "/usr/gnu/bin:/bin:/usr/bin:/sbin:/usr/sbin:/etc:/etc/bash"
 #endif
 
 /* Default primary and secondary prompt strings. */
@@ -91,20 +91,20 @@
 #define DEFAULT_BASHRC "~/.bashrc"
 
 /* System-wide .bashrc file for interactive shells. */
-/* #define SYS_BASHRC "/etc/bash.bashrc" */
+#define SYS_BASHRC "/etc/bash/bashrc"
 
 /* System-wide .bash_logout for login shells. */
-/* #define SYS_BASH_LOGOUT "/etc/bash.bash_logout" */
+#define SYS_BASH_LOGOUT "/etc/bash/bash_logout"
 
 /* Define this to make non-interactive shells begun with argv[0][0] == '-'
    run the startup files when not in posix mode. */
-/* #define NON_INTERACTIVE_LOGIN_SHELLS */
+#define NON_INTERACTIVE_LOGIN_SHELLS 1
 
 /* Define this if you want bash to try to check whether it's being run by
    sshd and source the .bashrc if so (like the rshd behavior).  This checks
    for the presence of SSH_CLIENT or SSH2_CLIENT in the initial environment,
    which can be fooled under certain not-uncommon circumstances. */
-/* #define SSH_SOURCE_BASHRC */
+#define SSH_SOURCE_BASHRC
 
 /* Define if you want the case-capitalizing operators (~[~]) and the
    `capcase' variable attribute (declare -c). */
